packages/openactive-integration-tests/test/features/payment/common.js

```diff
@@ -73,6 +73,9 @@ function successTests(expectedPrepayment, bookReqTemplateRef) {
     FlowStageUtils.describeRunAndCheckIsSuccessfulAndValid(bookRecipe, () => {
       if (bookRecipe.p) {
         itShouldHavePrepayment(expectedPrepayment, () => bookRecipe.p.getOutput().httpResponse.body);
+        it('should have state: deleted', () => {
+          expect(bookRecipe.orderFeedUpdateAfterDelete.getOutput().httpResponse.body.state).to.equal('deleted');
+        });
```

Comment:
And then why is it the successful payment test which is checking for this deleted item?

Reply:
Perhaps it should be in one of the `minimal-proposal` tests?

Reply:
You could do this by splitting out `FlowStageRecipes.book` into ``FlowStageRecipes.bookSimple` and `FlowStageRecipes.bookApproval`. `book` just returns one or the other depending on the order item criteria given to it.

This way, you can have the minimal-proposal tests use `FlowStageRecipes.bookApproval`, which means that they get the benefit of using the book recipe, which includes things like this new Order Feed update listener while still being explicit about only using the approval flow

Reply:
ah maybe I wasn't following our chat earlier... the test for the delete was to check that the proposal got deleted... which should be checked every time?